packages/shared/src/components/onboarding/OnboardingPlusStep.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC, useState } from 'react';
+import classNames from 'classnames';
+import { useViewSize, ViewSize } from '../../hooks';
+import {
+  ProductOption,
+  usePaymentContext,
+} from '../../contexts/PaymentContext';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { PlusComparingCards } from '../plus/PlusComparingCards';
+
+interface OnboardingStepProps {
+  onClickNext: () => void;
+}
+
+const PlusBillingCycleSwitch: FC<{
+  productOptions: ProductOption[];
+  currentCycleIndex: number;
+  onChangeCycle: (index: number) => void;
+}> = ({ productOptions, currentCycleIndex, onChangeCycle }) => {
+  return (
+    <div
+      aria-label="Select billing cycle"
+      role="radiogroup"
+      className="mx-auto my-6 inline-flex gap-1 rounded-12 border border-border-subtlest-tertiary p-1 tablet:my-8"
+    >
+      {productOptions.map(({ label, extraLabel }, index) => {
+        const isActive = index === currentCycleIndex;
+        const variant = isActive ? ButtonVariant.Float : ButtonVariant.Option;
+        return (
+          <Button
+            aria-checked={isActive}
+            aria-label={label}
+            className="min-w-24 justify-center"
+            key={label}
+            onClick={() => onChangeCycle(index)}
+            role="radio"
+            size={ButtonSize.Medium}
+            variant={variant}
+          >
+            <span>
+              <Typography
+                tag={TypographyTag.Span}
+                type={TypographyType.Callout}
+                color={TypographyColor.Primary}
+                className={classNames('text-center capitalize', {
+                  'font-normal': !isActive,
+                })}
+              >
+                {label}
+              </Typography>
+              {extraLabel && (
+                <Typography
+                  tag={TypographyTag.Span}
+                  type={TypographyType.Caption2}
+                  color={TypographyColor.StatusSuccess}
+                  className="block text-center font-normal"
```

Comment:
You shouldn't need normal here right if bold is not passed?

Reply:
Bold coming from button outside, so i need to override it for not making it inherited